service-bus.tf

```diff
@@ -0,0 +1,22 @@
+locals {
+  events_topic_name         = "${var.product}-events-topic-${var.env}"
+  servicebus_namespace_name = "${var.product}-servicebus-${var.env}"
+  resource_group_name       = azurerm_resource_group.rg.name
+}
+
+module "servicebus-namespace" {
+  source              = "git@github.com:hmcts/terraform-module-servicebus-namespace?ref=master"
+  name                = local.servicebus_namespace_name
+  location            = var.location
+  resource_group_name = azurerm_resource_group.rg.name
+  env                 = var.env
+  common_tags         = local.tags
+  sku                 = "Premium"
```

Comment:
when we enable Premium do we have by default High Availability? or we need to explicitly configure for HA?

Reply:
`  zoneRedundant       = true` now @mario-paniccia

Reply:
Could we make this configurable per environment? I guess on Demo for example we won't need `zoneRedundant`?

Reply:
we can remove this var in the specific inf branches